### PR TITLE
wait for kotsadm and kotsadm-api deployments to be ready before continuing

### PR DESCRIPTION
**Pull request by laverya**

### CODE REVIEW

File: addons/kotsadm/1.14.0-beta.2/install.sh [hunk 1]
```diff
@@ -344,3 +347,31 @@ function kotsadm_namespaces() {
         kubectl create ns "$NAMESPACE" 2>/dev/null || true
     done
 }
+
+function kotsadm_health_check() {
+    if [[ -n $(kubectl get pods -l app=kotsadm -o jsonpath="{range .items[*]}{range .status.conditions[*]}{ .type }={ .status }{'\n'}{end}{end}" | grep Ready=False) ]]; then
```
Comment (emosbaugh) [inline addons/kotsadm/1.14.0-beta.2/install.sh]: What if there are no pods?

Comment (emosbaugh) [inline addons/kotsadm/1.14.0-beta.2/install.sh]: I don’t think you need the bash style double bracket here

Comment (laverya) [reply addons/kotsadm/1.14.0-beta.2/install.sh]: possibly; I copied this from the weave pod health check code

Comment (laverya) [reply addons/kotsadm/1.14.0-beta.2/install.sh]: then we get the current 'no spinner' behavior, I suppose

Comment (laverya) [reply addons/kotsadm/1.14.0-beta.2/install.sh]: but pod creation should follow the deployment creation very quickly

**Pull request merged**